README.md

```diff
@@ -58,7 +58,7 @@ scrape_configs:
   scrape_interval: 5s
   honor_labels: true
   static_configs:
-    # EMQX IP address and port
+    # a list of addresses of all EMQX nodes
```

Comment:
Must all of EMQX nodes ？

Reply:
Yeap, it isn't same as the dashboard API, the metrics return by Prometheus API is independent